.gitignore

```diff
@@ -16,6 +16,8 @@ test/module/test-file
 !test/module/node_modules
 !test/module/**/*.js
 
+misc/*.js
```

Comment:
we do not need to put the `misc` folder into code base

Reply:
fixed